packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -589,9 +665,25 @@ export class SummarizerNode implements IRootSummarizerNode {
     /**
      * Tells whether summary tracking is in progress. True if "startSummary" API is called before summarize.
      */
-    protected isTrackingInProgress(): boolean {
+    public isSummaryInProgress(): boolean {
         return this.wipReferenceSequenceNumber !== undefined;
     }
+
+    /**
+     * Creates and throws an error due to unexpected conditions.
+     */
+    protected throwUnexpectedError(eventProps: ITelemetryErrorEvent): never {
+        const error = new LoggingError(eventProps.eventName, {
```

Comment:
Should (at least some cases) be using UsageError?
I.e. who is going to catch these errors?

Reply:
We don't know if these are because of usage or not. From summarizer node's perspective, summary / GC was not run on a node. It's not in a position to tell what caused it. This error is supposed to tell us that so that.
This will be caught during summarization process which will fail.

Reply:
Remove the commit that made changes to the summarizer nodes because they are not important for the release 3.0 branch.